Sources/SwiftInspectorCore/Tests/PropertyAnalzyerSpec.swift

```diff
@@ -0,0 +1,468 @@
+          ]
+          expect(propSet) == expectedPropSet
+        }
+      }
+
+      context("when there is a static property") {
+        beforeEach {
+          let content = """
+          public final class FakeType {
+            public static var thing: String = "Hello, World"
+          }
+          """
+          fileURL = try? Temporary.makeFile(content: content)
+        }
+
+        it("detects the property") {
+          let result = try? sut.analyze(fileURL: fileURL)
+          expect(result?.properties) == [
+            TypeWithPropInfo.PropInfo(
+              name: "thing",
+              typeAnnotation: "String",
+              comment: "",
+              access: .public,
+              scope: .static)
+          ]
+        }
+      }
+
+      context("when there is a private property") {
+        beforeEach {
+          let content = """
+          public final class FakeType {
+            private static var thing: String = "Hello, World"
+          }
+          """
+          fileURL = try? Temporary.makeFile(content: content)
+        }
+
+        it("detects the property") {
+          let result = try? sut.analyze(fileURL: fileURL)
+          expect(result?.properties) == [
+            TypeWithPropInfo.PropInfo(
+              name: "thing",
+              typeAnnotation: "String",
+              comment: "",
+              access: .private,
+              scope: .static)
+          ]
+        }
+      }
+
+      context("when there is a fileprivate property") {
+        beforeEach {
+          let content = """
+          public final class FakeType {
+            fileprivate static var thing: String = "Hello, World"
+          }
+          """
+          fileURL = try? Temporary.makeFile(content: content)
+        }
+
+        it("detects the property") {
+          let result = try? sut.analyze(fileURL: fileURL)
+          expect(result?.properties) == [
+            TypeWithPropInfo.PropInfo(
+              name: "thing",
+              typeAnnotation: "String",
+              comment: "",
+              access: .fileprivate,
+              scope: .static)
+          ]
+        }
+      }
+
+      context("when there is no type annotation") {
+        beforeEach {
+          let content = """
+          public final class FakeType {
+            private static var thing = "Hello, World"
+          }
+          """
+          fileURL = try? Temporary.makeFile(content: content)
+        }
+
+        it("detects the property with no type information") {
+          let result = try? sut.analyze(fileURL: fileURL)
+          expect(result?.properties) == [
+            TypeWithPropInfo.PropInfo(
+              name: "thing",
+              typeAnnotation: nil,
+              comment: "",
+              access: .private,
+              scope: .static)
+          ]
+        }
+      }
+
+      context("when there is a line comment") {
```

Comment:
I don't know the inner workings of SwiftSyntax well enough but would we want to test a multi-line comment? @fdiaz you know better than me if that's getting into testing the internals of SwiftSyntax.

Reply:
I can certainly test it, I just didn't think it was an interesting or useful test given the code